src/pages/experiments/[experimentId]/data-processing/index.jsx

```diff
@@ -235,16 +235,16 @@ const DataProcessingPage = ({ experimentId, experimentData }) => {
       key: 'doubletScores',
       name: getUserFriendlyQCStepName('doubletScores'),
       description:
-        <span>
-          Droplets may contain more than one cell.
-          In such cases, it is not possible to distinguish which reads came from which cell.
-          Such “cells” cause problems in the downstream analysis as they appear as an intermediate type.
-          “Cells” with a high probability of being a doublet should be excluded.
-          The probability of being a doublet is calculated using ‘scDblFinder’.
-          For each sample, the default threshold tries to minimize both the deviation in the
-          expected number of doublets and the error of a trained classifier. For more details see
+  <span>
+    Droplets may contain more than one cell.
+    In such cases, it is not possible to distinguish which reads came from which cell.
+    Such “cells” cause problems in the downstream analysis as they appear as an intermediate type.
+    “Cells” with a high probability of being a doublet should be excluded.
+    The probability of being a doublet is calculated using ‘scDblFinder’.
+    For each sample, the default threshold tries to minimize both the deviation in the
+    expected number of doublets and the error of a trained classifier. For more details see
     {' '}
-          <a href='https://bioconductor.org/packages/devel/bioc/vignettes/scDblFinder/inst/doc/scDblFinder.html#thresholding' rel='noreferrer' target='_blank'>scDblFinder thresholding</a>
+    <a href='https://bioconductor.org/packages/devel/bioc/vignettes/scDblFinder/inst/doc/scDblFinder.html#thresholding' rel='noreferrer' target='_blank'>scDblFinder thresholding</a>
```

Comment:
this looks miss-aligned

Reply:
thats how the linter moves it

Reply:
its not missaligned, the block above it is also shifted 1 tab back